python/GafferArnoldTest/ArnoldTextureBakeTest.py

```diff
@@ -310,6 +311,7 @@ def testTasks( self ):
 			self.assertGreater( l, 2 )
 			self.assertLess( l, 8 )
 
+	@unittest.skipIf( "TRAVIS" in os.environ or "TF_BUILD" in os.environ, "Arnold license not available" )
```

Comment:
Found this a little confusing at first read, but then other skip messages don't seem to mention 'as were in cloud build' either...